dockstore-common/src/main/java/io/dockstore/common/yaml/YamlNotebook.java

```diff
@@ -49,12 +49,18 @@ public void setName(final String name) {
         this.name = name;
     }
 
+    /**
+     * Get the format (i.e. IPYNB) of the notebook file.
```

Comment:
I think it should be `e.g.` not `i.e.` :) Don't change.

Reply:
(it is)
e.g. -> for example
i.e. -> in other words